src/Traits/AttributesUtils.php

```diff
@@ -13,6 +13,11 @@ public function mergeOptions($element, $options = [])
             unset($options['class']);
         }
 
+        if (isset($options['placeholder'])) {
+            $newElement = $newElement->placeholder($options['class']);
+            unset($options['placeholder']);
+        }
```

Comment:
Hi, thanks for your contribution, I understand that it change adds support for the placeholder, but you are calling to the `'class'` key of options instead `'placeholder'`. Do you think that it could be a mistake?. Here a little suggestion, if you are agree I can merge your contribution :)

```suggestion
        if (isset($options['placeholder'])) {
            $newElement = $newElement->placeholder($options['placeholder']);
            unset($options['placeholder']);
        }
```